src/ngraph/op/broadcast.cpp

```diff
@@ -141,9 +142,33 @@ void op::v1::Broadcast::validate_and_infer_types()
     PartialShape result_shape{PartialShape::dynamic()};
     if (input_value(1).get_node_shared_ptr()->is_constant())
     {
-        result_shape = static_pointer_cast<op::Constant>(input_value(1).get_node_shared_ptr())
+        result_shape = static_pointer_cast<op::v0::Constant>(input_value(1).get_node_shared_ptr())
                            ->get_shape_val();
     }
+    else if (dynamic_pointer_cast<op::v0::Concat>(input_value(1).get_node_shared_ptr()))
```

Comment:
This test will never be true since you caught all the constants in the previous test.

Reply:
Done